src/components/Header/Nav/navBar.module.css

```diff
@@ -96,7 +96,7 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  z-index: 10;
+  z-index: 9999;
```

Comment:
there is no reason to put something so high. Just go in increments of 100 unless necessary for other reason (if so add a comment)